chat-plugins/info.js

```diff
@@ -10,8 +10,7 @@
  * @license MIT license
  */
 
-var commands = exports.commands = {
```

Comment:
I thought setting this variable was only used with how the command-parser specifically imported `config/commands.js` ? I don't fully understand what it's there for, so I'll put it back.